example/home/test/test_blog.py

```diff
@@ -915,7 +916,6 @@ def test_blog_page_tags(self):
         tags = executed["data"]["page"]["tags"]
         self.assertEqual(len(tags), 3)
         for idx, tag in enumerate(tags, start=1):
-            self.assertEqual(int(tag["id"]), idx)
```

Comment:
question: why remove it? (I suspect I know the answer, but best not to assume)

Reply:
There is no guarantee that tags are ordered by ID or that a given database's auto incremented will start the ids at 0 and advance by 1. This is simply a frail assertion that doesn't prove anything about the validity of the data returned by the database.